Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -187,26 +198,49 @@ void MemoryViewWidget::Update()
       });
       break;
     case Type::Float32:
-      update_values(
-          [&accessors](u32 address) { return QString::number(accessors->ReadF32(address)); });
+      update_values([&accessors](u32 address) {
+        QString string = QString::number(accessors->ReadF32(address), 'g', 4);
+        // Align to first digit.
+        if (!string.startsWith(QLatin1Char('-')))
+          string.prepend(QLatin1Char(' '));
```

Comment:
This relies on the font being fixed-width, but to be fair I don't know a better method either.

Reply:
Good point.  I don't think there's a method for non-fixed. I'd expect people to use fixed width debug fonts though.